### DISCUSSION TITLE
Selectively show Panzoom controls

### DISCUSSION THREAD

**Discussion by pbowyer** [fancyapps/ui#363] [Q&A]
I'm using Fancybox/Carousel/Panzoom on a lot of images (thank you, works well!) and I have a special type of image where I want to show the zoom in/out controls (plus a 'full screen control' which I will need to add to the plugin) along with other customised settings.

Per https://github.com/fancyapps/ui/discussions/58 I tried `Panzoom.Plugins = {Controls};` which works, but this adds the controls to all lightbox images.

I tried adding the plugin to the Panzoom options, but this isn't picked up:
```js
Fancybox.bind('[data-fancybox-controls]', {
    Panzoom: {
        Plugins: {
            Controls,
        }
    }
}
```

How can I activate the controls selectively?

**Comment by fancyapps** [comment]
Hi,

Well, good question. I'll try to figure out how to make it easy to implement.

For now, you could create and append your own buttons inside the slide within `load` event. Then you could add click handler and use Panzoom API, for example:

```JS
Fancybox.getInstance().getSlide().Panzoom.toggleZoom();
```
